### PR TITLE
Feature/readme improvements

### PR DESCRIPTION
**Pull request by andreluizmachado**
- Added Installation instructions Section
- Added Set up deliveries on TAO Platform Section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,57 @@
...
+# TAO _taoPublishing_ extension
 
-Pushes a test to a delivery environment
+![TAO Logo](https://github.com/oat-sa/taohub-developer-guide/raw/master/resources/tao-logo.png)
+
+![GitHub](https://img.shields.io/github/license/oat-sa/extension-tao-publishing.svg)
+![GitHub release](https://img.shields.io/github/release/oat-sa/extension-tao-publishing.svg)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/y/oat-sa/extension-tao-publishing.svg)
+
+> Pushes a test to a delivery environment
```
Comment (luisfn) [inline README.md]: It's only a test?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+$ docker container exec -it {YOUR-TAO-INSTANCE-PHP-CONTAINER} php tao/scripts/installExtension.php taoPublishing
+```
+
+After the install just log out and log in again on the TAO Platform.
+
+As a system administrator you can also install it through the TAO Extension Manager:
+- Settings (the gears on the right hand side of the menu) -> Extension manager
+- Select _taoPublishing_ on the right hand side, check the box and hit _install_
+
+##Set up deliveries on TAO Platform
```
Comment (luisfn) [inline README.md]: I think you need and extra blank line because it's showing `##`

File: README.md (hunk 1 above)
Comment (andreluizmachado) [reply README.md]: Thanks. I'll try to explain better.

File: README.md (hunk 2 above)
Comment (andreluizmachado) [reply README.md]: Fixed.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,57 @@
...
+
+After the install just log out and log in again on the TAO Platform.
+
+As a system administrator you can also install it through the TAO Extension Manager:
+- Settings (the gears on the right hand side of the menu) -> Extension manager
+- Select _taoPublishing_ on the right hand side, check the box and hit _install_
+
+## Set up deliveries on TAO Platform
+
+To be able to synchronize, you need to set up the platforms that you want to sync. You can do it by:
```
Comment (tikhanovichA) [inline README.md]: What about `Publish Actions`. As I remember this is the list of actions for triggering publishing to a remote env

Comment (andreluizmachado) [reply README.md]: I forgot to mention it. I'll do that soon. Many thanks.

Comment (andreluizmachado) [reply README.md]: Hi @tikhanovichA. I checked in the code by removing the config on the 
config/taoPublishing/PublishingService.conf.php and config/default/PublishingService.conf.php and the behavior that I got was that the extension just removes the event from the admin platform configuration but doesn't` mean that the sync action will be removed. For example, if you remove the DeliveryCreatedEvent from the config, it'll remove it from the admin platform form, but the user can still sync it when creating a new delivery.
So, in this case, this is not something that we can set up actually. That's why I wouldn't put anything related to this on the readme.

Comment (tikhanovichA) [reply README.md]: it looks like a bug.
So, if I will not use that block with `Publish Actions`, how taoPublishing will work?

Comment (andreluizmachado) [reply README.md]: Hello @tikhanovichA. After more investigation, I could find more details about how to set up the Publish Actions. Could you please take a look at the README changes again, please?

**Pull request merged**